channels/sessions.py

```diff
@@ -225,12 +224,15 @@ async def send(self, message):
         # Pass up the send
         return await self.real_send(message)
 
-    def save_session(self):
+    async def save_session(self):
         """
         Saves the current session.
         """
         try:
-            self.scope["session"].save()
+            if django.VERSION >= (5, 1):
+                await self.scope["session"].asave()
+            else:
+                database_sync_to_async(self.scope["session"].save)()
```

Comment:
Missing `await`?

Reply:
Oops yep, I’ll fix this next time I’m at a computer

Reply:
I'm guess we're missing test coverage here 🤔

Reply:
Good man! (Happy to pull that in already as a "Improve coverage" type thing 🎁)